pytest_mozwebqa/cloud/browserstack.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ConfigParser import ConfigParser
+import os
+
+import requests
+from selenium import webdriver
+
+
+class BrowserStack(object):
+
+    name = 'BrowserStack'
+
+    def __init__(self):
+        username = None
+        access_key = None
+
+        config = ConfigParser()
+        config.read('setup.cfg')
+
+        section = 'browserstack'
+        if config.has_section(section):
+            if config.has_option(section, 'username'):
+                username = config.get(section, 'username')
+            if config.has_option(section, 'access-key'):
+                access_key = config.get(section, 'access-key')
+
+        self.username = os.getenv('BROWSERSTACK_USERNAME', username)
```

Comment:
This means that the environment variables, if they exist, will always override any values specified in `setup.cfg`, correct? I think this is fine, as I can't imagine a time when you'd have the environment variables and still want to use the values in  `setup.cfg`, but I wanted to make sure this was intentional. I suppose another option would be to only check for environment variables if the values are not found or specified in  `setup.cfg`, but then you end up with the opposite problem.